### PR TITLE
CSP header

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
Fixing: https://github.com/mozilla/nsf-challenge/issues/31

**Comment by ScottDowne** [comment]
@cadecairos part of me thinks we can let the broken live-server script be there, and fix later.

It's not an issue on production, and it doesn't break any necessary functionality for dev.

**Comment by april** [comment]
This seems to work for me:

```
meta(http-equiv=`Content-Security-Policy`, content=`default-src 'none'; connect-src ws://127.0.0.1:2017; font-src https://code.cdn.mozilla.net; img-src 'self' data:; script-src 'self' 'sha256-iN7wpJdxHlpujRppkOA8N0+Mzp0ZqZr3lCtxM00Y63c='; style-src 'self' https://code.cdn.mozilla.net;`)
```

It will break if the liveserver injected code ever changes, but all you would need to do is update the 'sha256' hash.

File: source/pug/partials/header.pug [hunk 1]
```diff
@@ -1,6 +1,7 @@
 doctype html
 html
   head
+    meta(http-equiv=`Content-Security-Policy`, content=`default-src 'none'; connect-src ws://127.0.0.1:2017; font-src https://code.cdn.mozilla.net; img-src 'self' data:; script-src 'self' 'sha256-iN7wpJdxHlpujRppkOA8N0+Mzp0ZqZr3lCtxM00Y63c='; style-src 'self' https://code.cdn.mozilla.net;`)
```
Comment (cadecairos) [inline source/pug/partials/header.pug:4]: Works like a charm with the hashed script directive.

**Review by cadecairos — APPROVED** [review]

**Pull request merged**